spec/factories.rb

```diff
@@ -0,0 +1,35 @@
+FactoryGirl.define do
+  factory :user do
+    first_name "Alice"
+    last_name "Smith"
+    display_name "valid"
+    email "alica@email.com"
+    about_me "I am Alice"
+    password "password"
+  end
+
+  factory :category do
+    name "Condo"
+  end
+
+  factory :listing do
+    before(:create) do |listing|
+      listing.category = create(:category)
+    end
+
+    before(:create) do |listing|
+      listing.user = create(:user, display_name: "validd", email: "asmith@emial.com")
```

Comment:
I'd love to hear how factory girl works - thought it could create multiple instances of listings that belong to a user?  
also - spelling? (display_name: "validd", email: "asmith@emial.com")